### PR TITLE
bugfix/213-peer-dependency

### LINKED ISSUE #213: Peer dependency for Angular 10
### Requested feature description
Can Angular 10 be added as a valid peer dependency? or possibly >= 9? unless you plan on updating this package more frequently to keep dependencies synced with Angular release cadence.

Thanks

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed #213, changed peer dependencies.

### CODE REVIEW

**Comment by hakimio** [comment]
@karolkolodziej 
Since @KacperMadej explicitely set it the dependency to `^9.0.0`, I am pretty sure it should be `>=9.0.0`. 
Have you actually tested this package with really old Angular versions? I am pretty sure it won't correctly with Angular v2.

**Comment by mateuszkornecki** [comment]
> @karolkolodziej
> Since @KacperMadej explicitely set the dependency to `^9.0.0`, I am pretty sure it should be `>=9.0.0`.
> Have you actually tested this library with really old Angular versions? I am pretty sure it won't work correctly with Angular v2.

@hakimio You are right, we are not providing support over angular v2, however, the library is working correctly with angular 6. Sorry for the late reply, but it seems that your comment was stuck in limbo (I must have missed the notification).

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "@angular/router": "10.0.8",
     "@highcharts/map-collection": "^1.1.3",
     "core-js": "^3.6.4",
-    "highcharts": "^8.1.2",
+    "highcharts": "^8.2.2",
```
Comment (hakimio) [inline package.json:28]: Seems like unrelated change.

Comment (karolkolodziej) [reply package.json:28]: Yes, it doesn't change anything in the wrapper itself. 
This changes only the `higcharts` version in the demos which are attached to the repository.

Comment (hakimio) [reply package.json:28]: Maybe demo dependency update should be in a different PR?

**Review by mateuszkornecki — APPROVED** [review]
